src/js/profile/indicator.js

```diff
@@ -21,74 +21,81 @@ export class Indicator extends Component {
 
         isLast = _isLast;
 
-        if (indicatorData.chartConfiguration.rawHtml) {
-            this.addIndicatorText(wrapper, title, indicatorData);
-        }
-        else {
-            this.addIndicatorChart(wrapper, title, indicatorData, detail);
+        this.indicatorNode = indicatorClone.cloneNode(true);
+        $(indicatorTitleClass, this.indicatorNode).text(this.title);
+        $(chartDescClass, this.indicatorNode).text(this.indicatorData.description);
+        this.extractMetadata(this.indicatorData.metadata, this.indicatorNode);
+    }
+
+    extractMetadata(metadata, indicatorNode) {
```

Comment:
@jbothma will the info block have metadata? I don't think so right?

Reply:
It would be nice of this had a faint description and source link and label like profile indicators have. Is that what you're referring to?

Reply:
I was thinking of the TextBlock as a generic text block so that admins could add arbitrary text